docs/input4MIPs_source-id_CMIP6Plus.html

```diff
@@ -22,20 +22,26 @@
 </head>
 <body>
 <p><h1>Input4MIPs CMIP6Plus source IDs: v6.5.20a1</h1><p>
```

Comment:
as a nit, I always like to use "input" rather than "Input" but now I am devolving into pedantry

Reply:
Easy fix, will make it here too